src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -100,15 +171,31 @@
         theme-dark        (subscribe [:theme/dark])
         remote-graph-conf (subscribe [:db/remote-graph-conf])
         socket-status     (subscribe [:socket-status])
+        win-focused?    (subscribe [:win-focused?])
+        win-maximized?    (subscribe [:win-maximized?])
+        win-fullscreen?    (subscribe [:win-fullscreen?])
         merge-open?       (reagent.core/atom false)]
     (fn []
       [:<>
 
-
        (when @merge-open?
          [filesystem/merge-modal merge-open?])
 
-       [:header (use-style app-header-style)
+       [:header (use-style app-header-style
+                           {:class (vec (flatten
```

Comment:
I'm not sure you need the `flatten`.

something like this might also work:

`{:class [(if @theme-dark "theme-dark" "theme-light") (if electron? ...)]}`

I'd expect `nil`s to be ignored.